src/Client.php

```diff
@@ -342,6 +273,10 @@ public function call($method, $params = [])
 
         $this->doRequest($request);
 
+        if ($this->lastResponse === null) {
+            return null;
+        }
+
```

Comment:
```suggestion
        assert($this->lastResponse !== null);
```

We can guarantee it's not null after a successful call to `doRequest()`